src/main/java/com/yelp/nrtsearch/server/luceneserver/QueryNodeMapper.java

```diff
@@ -185,6 +188,15 @@ private FunctionScoreQuery getFunctionScoreQuery(
         scriptFactory.newFactory(params, state.docLookup));
   }
 
+  private FunctionMatchQuery getFunctionFilterQuery(
+      FunctionFilterQuery functionFilterQuery, IndexState state) {
+    ScoreScript.Factory scriptFactory =
+        ScriptService.getInstance().compile(functionFilterQuery.getScript(), ScoreScript.CONTEXT);
+    Map<String, Object> params =
+        ScriptParamsUtils.decodeParams(functionFilterQuery.getScript().getParamsMap());
+    return new FunctionMatchQuery(scriptFactory.newFactory(params, state.docLookup), x -> x > 0);
+  }
+
```

Comment:
seems FunctionFilterQuery.getQuery is not used? We can delete query field if it is not used